Lib/fontTools/misc/psCharStrings.py

```diff
@@ -980,6 +980,7 @@ def draw(self, pen):
 				self.private.nominalWidthX, self.private.defaultWidthX,
 				self.private)
 		extractor.execute(self)
+		extractor.endPath()
```

Comment:
You call endPath, not closePath as the title says

Reply:
Fixed the title.